tests/pytest_framework/src/syslog_ng/syslog_ng_cli.py

```diff
@@ -88,7 +109,7 @@ def reload(self, config):
         self.__syslog_ng_ctl.reload()
 
         # wait for reload and check reload result
-        if not self.__syslog_ng_ctl.wait_for_control_socket_alive():
+        if not self.wait_for_control_socket_alive():
```

Comment:
This will not needed, but the new __wait_for_process_alive()

Reply:
as mentioned above, wait_for_process_alive cannot help us now.